static/app/views/monitors/components/detailsSidebar.tsx

```diff
@@ -0,0 +1,146 @@
+}
+
+export default function DetailsSidebar({monitorEnv, monitor}: Props) {
+  const {checkin_margin, schedule, schedule_type, max_runtime, timezone} = monitor.config;
+
+  const slug = (
+    <Clipboard value={monitor.slug}>
+      <MonitorSlug>
+        <SlugText>{monitor.slug}</SlugText>
+        <IconCopy size="xs" />
+      </MonitorSlug>
+    </Clipboard>
+  );
+
+  return (
+    <React.Fragment>
+      <CheckIns>
+        <SectionHeading>{t('Last Check-In')}</SectionHeading>
+        <SectionHeading>{t('Next Check-In')}</SectionHeading>
+        <div>
+          {monitorEnv?.lastCheckIn ? (
+            <TimeSince
+              unitStyle="regular"
+              liveUpdateInterval="second"
+              date={monitorEnv.lastCheckIn}
+            />
+          ) : (
+            '-'
+          )}
+        </div>
+        <div>
+          {monitorEnv?.nextCheckIn ? (
+            <TimeSince
+              unitStyle="regular"
+              liveUpdateInterval="second"
+              date={monitorEnv.nextCheckIn}
+            />
+          ) : (
+            '-'
+          )}
+        </div>
+      </CheckIns>
+      <SectionHeading>{t('Schedule')}</SectionHeading>
+      <Schedule>
+        <MonitorIcon status={MonitorStatus.OK} size={12} />
+        <Text>
+          {schedule_type === ScheduleType.CRONTAB
+            ? schedule
+            : getIntervalScheduleText(schedule as IntervalConfig['schedule'])}
+        </Text>
+        <MonitorIcon status={MonitorStatus.MISSED_CHECKIN} size={12} />
+        <Text>
+          {defined(checkin_margin)
+            ? tn('Check-ins %s min late', 'Check-ins %s mins late', checkin_margin)
+            : t('Check-ins that are late')}
+        </Text>
+        <MonitorIcon status={MonitorStatus.ERROR} size={12} />
+        <Text>
+          {tn(
+            'Check-ins longer than %s min or errors',
+            'Check-ins longer than %s mins or errors',
+            max_runtime ?? DEFAULT_MAX_RUNTIME
+          )}
+        </Text>
+      </Schedule>
+      <SectionHeading>{t('Cron Details')}</SectionHeading>
+      <KeyValueTable>
+        <KeyValueTableRow keyName={t('Monitor Slug')} value={slug} />
+        {schedule_type === ScheduleType.CRONTAB && (
+          <KeyValueTableRow keyName={t('Timezone')} value={timezone} />
+        )}
+        <KeyValueTableRow
+          keyName={t('Date created')}
+          value={getFormattedDate(monitor.dateCreated, 'MMM D, YYYY')}
+        />
+      </KeyValueTable>
+    </React.Fragment>
+  );
+}
+
+const CheckIns = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin-bottom: ${space(2)};
+`;
+
+const Schedule = styled('div')`
+  display: grid;
+  grid-template-columns: max-content 1fr;
+  margin-bottom: ${space(2)};
+  align-items: center;
+  gap: ${space(1)};
+`;
+
+const MonitorSlug = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr max-content;
+  align-items: center;
+  gap: ${space(0.5)};
+  cursor: pointer;
```

Comment:
Why pointer?

Reply:
just to show that it's clickable and copyable

<img width="341" alt="image" src="https://user-images.githubusercontent.com/9372512/235225955-b98e1da0-05fb-4dff-b155-4a207e7b07dc.png">

Reply:
Do we do the same thing in other places?

Reply:
Why doesn't the clipboard component do this for us actually?

Reply:
seems like a bit of the same thing in other places https://github.com/getsentry/sentry/blob/e4ac987d6bc1f7f196d332699f56e5093ea5567b/static/app/components/version.tsx#L185

not sure why clipboard doesn't do this for us though

Reply:
lol yea this is just everywhere, https://github.com/getsentry/sentry/blob/952e4de9a068694babaf795c2949fedbdd1ba606/static/app/components/events/userFeedback.tsx#L70